dapps/assets/.gitignore

```diff
@@ -0,0 +1,39 @@
+# Node
+node_modules/*
+npm-debug.log
+
+# TypeScript
+src/*.js
```

Comment:
why is there even.js files and if yes, maybe we need them?

Reply:
I don't think there are any. It's just a standard gitignore template I guess.